patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1491,13 +1482,13 @@
  				}
  
  				if (armor[k].prefix == 73)
-@@ -9264,6 +_,22 @@
+@@ -9264,7 +_,25 @@
  
  				if (armor[k].prefix == 80)
  					meleeSpeed += 0.04f;
 +
 +				ItemLoader.UpdateEquip(armor[k], this);
-+			}
+ 			}
```

Comment:
accidental indentation change? or fix?

Reply:
Fix, I believe. Instead of us adding the closing '}', we now use the existing vanilla '}'.
I'll take another look for any that aren't a fix.